images/win/scripts/Installers/Validate-Awscli.ps1

```diff
@@ -0,0 +1,26 @@
+################################################################################
+##  File:  Validate-awscli.ps1
+##  Desc:  Validate awscli
+################################################################################
+
+$command = Get-Command -Name 'awscli'
+
+if ($command)
+{
+    Write-Host "awscli on path"
+}
+else
+{
+    Write-Host 'awscli is not on path'
+    exit 1
+}
+
+# Adding description of the software to Markdown
+$SoftwareName = "Aws CLI"
+$version = $command.Version.ToString()
```

Comment:
```
PS > (Get-Command -Name 'aws').Version.ToString()
0.0.0.0
```

Parse awscli version from `aws --version` output:
`aws-cli/2.0.5 Python/3.7.5 Windows/10 botocore/2.0.0dev9`

Reply:
Let's cut this line to take only `aws-cli 2.0.5`